templates/full/src/controls/ui.tsx

```diff
@@ -53,6 +52,7 @@ export const models = {
   qrcode: QRCodeExample,
   webview: WebViewExample,
   video: VideoExample,
+  rn: ReactNativeExample,
```

Comment:
maybe we should have an example which shows the combination of `StackScreen` and `StatusBar` features? Although that example might need to be dynamic somehow either with server or client state.. just an idea, not important for this PR